sentinelhub/commands.py

```diff
@@ -2,14 +2,18 @@
 Module that implements command line interface for the package
 """
 
+from typing import Any, Callable, TypeVar
+
 import click
 
 from .config import SHConfig
 from .download import DownloadClient, DownloadRequest
 
+FC = TypeVar("FC", bound=Callable[..., Any])
```

Comment:
Should we make the name `FC` a bit more verbose to be clear what it stands for? Maybe `_FunctionType`?

Reply:
I used `FC` because that is what `click` uses. We can be more verbose if needed.

Reply:
Hm, in that case let's leave it as it is.